src/main/java/me/eccentric_nz/TARDIS/TARDISCommandSetter.java

```diff
@@ -68,83 +69,83 @@
+        Objects.requireNonNull(plugin.getCommand("tardis")).setExecutor(new TARDISCommands(plugin));
+        Objects.requireNonNull(plugin.getCommand("tardis")).setTabCompleter(new TARDISTabComplete(plugin));
+        Objects.requireNonNull(plugin.getCommand("tardisadmin")).setExecutor(new TARDISAdminCommands(plugin));
+        Objects.requireNonNull(plugin.getCommand("tardisadmin")).setTabCompleter(new TARDISAdminTabComplete(plugin));
         TARDISConfigCommand tardisConfigCommand = new TARDISConfigCommand(plugin);
         plugin.getGeneralKeeper().setTardisConfigCommand(tardisConfigCommand);
-        plugin.getCommand("tardisconfig").setExecutor(tardisConfigCommand);
-        plugin.getCommand("tardisconfig").setTabCompleter(new TARDISConfigTabComplete(plugin));
-        plugin.getCommand("tardisdev").setExecutor(new TARDISDevCommand(plugin));
-        plugin.getCommand("tardisdev").setTabCompleter(new TARDISDevTabComplete());
-        plugin.getCommand("tardisarea").setExecutor(new TARDISAreaCommands(plugin));
-        plugin.getCommand("tardisarea").setTabCompleter(new TARDISAreaTabComplete());
-        plugin.getCommand("tardisartron").setExecutor(new TARDISArtronStorageCommand(plugin));
-        plugin.getCommand("tardisartron").setTabCompleter(new TARDISArtronTabComplete());
-        plugin.getCommand("tardisbind").setExecutor(new TARDISBindCommands(plugin));
-        plugin.getCommand("tardisbind").setTabCompleter(new TARDISBindTabComplete());
-        plugin.getCommand("tardisbiome").setExecutor(new TARDISBiomeCommand(plugin));
-        plugin.getCommand("tardisbook").setExecutor(new TARDISBookCommands(plugin));
+        Objects.requireNonNull(plugin.getCommand("tardisconfig")).setExecutor(tardisConfigCommand);
+        Objects.requireNonNull(plugin.getCommand("tardisconfig")).setTabCompleter(new TARDISConfigTabComplete(plugin));
+        Objects.requireNonNull(plugin.getCommand("tardisdev")).setExecutor(new TARDISDevCommand(plugin));
+        Objects.requireNonNull(plugin.getCommand("tardisdev")).setTabCompleter(new TARDISDevTabComplete());
+        Objects.requireNonNull(plugin.getCommand("tardisarea")).setExecutor(new TARDISAreaCommands(plugin));
+        Objects.requireNonNull(plugin.getCommand("tardisarea")).setTabCompleter(new TARDISAreaTabComplete());
+        Objects.requireNonNull(plugin.getCommand("tardisartron")).setExecutor(new TARDISArtronStorageCommand(plugin));
+        Objects.requireNonNull(plugin.getCommand("tardisartron")).setTabCompleter(new TARDISArtronTabComplete());
+        Objects.requireNonNull(plugin.getCommand("tardisbind")).setExecutor(new TARDISBindCommands(plugin));
+        Objects.requireNonNull(plugin.getCommand("tardisbind")).setTabCompleter(new TARDISBindTabComplete());
+        Objects.requireNonNull(plugin.getCommand("tardisbiome")).setExecutor(new TARDISBiomeCommand(plugin));
+        Objects.requireNonNull(plugin.getCommand("tardisbook")).setExecutor(new TARDISBookCommands(plugin));
```

Comment:
I'm not really sure what the benefit of asserting non-null here is. It's dereferencing the nullable pointer in the same line, so this isn't making it fail any faster than it would've anyways. Is this just an IDE suggestion?

Reply:
That comment applies to _many_ other uses of `requireNotNull` throughout the code, I just picked a few of them for that comment

Reply:
I was sort of just obeying IntelliJ, yeah.

Reply:
You're better off doing what I did:

https://github.com/ChewMC/TransmuteIt/blob/master/src/main/java/pw/chew/transmuteit/TransmuteIt.java#L85-L113

Objects.requireNotNull() is generally frowned upon unless you're _absolutely sure_ you'll never get a null, otherwise it throws an exception, which it would anyway.